### DISCUSSION TITLE
Bug in sending email

### DISCUSSION THREAD

**Discussion by ducnhoxlatui** [Budibase/budibase#10951] [Help] [answered]
Hi guys, I tried to send email as a test in automation. It notified that it's sent but when i checked the mail which i use to receive mail, i can see nothing sent.
![image](https://github.com/Budibase/budibase/assets/71976420/41c1f923-b971-44ae-bab5-0dca851ac3ec)
![image](https://github.com/Budibase/budibase/assets/71976420/ffc7b5a2-aa7e-431e-b3a3-4263ff1420ea)
![image](https://github.com/Budibase/budibase/assets/71976420/90d82ad2-9806-4d1c-ac40-59f67297328b)

Thanks a lot!!!

**Comment by andz-bb** [comment]
Hey @ducnhoxlatui 

What is the output of your Send Email step in this case? Can you see any logs from your Mailjet account?

**Reply by ducnhoxlatui** [reply, marked as answer]
yah, 
solution: i have the solution that smtp i config need to be hosting by email such as student email or organization email because of the privacy of the hosting email not allow to host by gmail, yahoo or hot mail.
